src/navigations/root-stack.js

```diff
@@ -186,7 +186,7 @@ const AuthenticatedNavigator = () => {
         />
         <AuthenticatedStack.Screen
           name="OtherProfile"
-          component={withKeyboardWrapper(OtherProfile)}
+          component={withSafeAreaView(withKeyboardWrapper(OtherProfile))}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `withSafeAreaView` is a good move to ensure that the content is displayed correctly within the screen boundaries. However, it's important to note that if `withSafeAreaView` modifies any props or introduces new ones, you should make sure that `OtherProfile` and `withKeyboardWrapper` can handle those changes appropriately.
```diff
- component={withSafeAreaView(withKeyboardWrapper(OtherProfile))}
+ component={withSafeAreaView(withKeyboardWrapper(OtherProfile))}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->